boards.txt

```diff
@@ -5083,6 +5083,326 @@ lilygo_t3s3.menu.Revision.Radio_SX1280PA.build.variant=lilygo_t3_s3_sx1280pa
+twatchs3.build.f_cpu=240000000L
+twatchs3.build.flash_size=16MB
+twatchs3.build.flash_freq=80m
+twatchs3.build.flash_mode=dio
+twatchs3.build.boot=qio
+twatchs3.build.boot_freq=80m
+twatchs3.build.partitions=app3M_fat9M_16MB
+twatchs3.build.defines=-DBOARD_HAS_PSRAM -DARDUINO_T_WATCH_S3
+twatchs3.build.loop_core=
+twatchs3.build.event_core=
+twatchs3.build.psram_type=opi
+twatchs3.build.memory_type={build.boot}_{build.psram_type}
+
+## IDE 2.0 Seems to not update the value
+twatchs3.menu.JTAGAdapter.default=Disabled
+twatchs3.menu.JTAGAdapter.default.build.copy_jtag_files=0
+twatchs3.menu.JTAGAdapter.builtin=Integrated USB JTAG
+twatchs3.menu.JTAGAdapter.builtin.build.openocdscript=esp32s3-builtin.cfg
+twatchs3.menu.JTAGAdapter.builtin.build.copy_jtag_files=1
+
+twatchs3.menu.LoopCore.1=Core 1
+twatchs3.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+twatchs3.menu.LoopCore.0=Core 0
+twatchs3.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+twatchs3.menu.EventsCore.1=Core 1
+twatchs3.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+twatchs3.menu.EventsCore.0=Core 0
+twatchs3.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+twatchs3.menu.USBMode.hwcdc=Hardware CDC and JTAG
+twatchs3.menu.USBMode.hwcdc.build.usb_mode=1
+twatchs3.menu.USBMode.default=USB-OTG (TinyUSB)
+twatchs3.menu.USBMode.default.build.usb_mode=0
+
+twatchs3.menu.CDCOnBoot.default=Enabled
+twatchs3.menu.CDCOnBoot.default.build.cdc_on_boot=1
+twatchs3.menu.CDCOnBoot.cdc=Disabled
+twatchs3.menu.CDCOnBoot.cdc.build.cdc_on_boot=0
+
+twatchs3.menu.MSCOnBoot.default=Disabled
+twatchs3.menu.MSCOnBoot.default.build.msc_on_boot=0
+twatchs3.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+twatchs3.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+twatchs3.menu.DFUOnBoot.default=Disabled
+twatchs3.menu.DFUOnBoot.default.build.dfu_on_boot=0
+twatchs3.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+twatchs3.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+twatchs3.menu.UploadMode.default=UART0 / Hardware CDC
+twatchs3.menu.UploadMode.default.upload.use_1200bps_touch=false
+twatchs3.menu.UploadMode.default.upload.wait_for_upload_port=false
+twatchs3.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+twatchs3.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+twatchs3.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+
+twatchs3.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+twatchs3.menu.PartitionScheme.fatflash.build.partitions=ffat
+twatchs3.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+twatchs3.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
+twatchs3.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+twatchs3.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+twatchs3.menu.PartitionScheme.rainmaker=RainMaker
+twatchs3.menu.PartitionScheme.rainmaker.build.partitions=rainmaker
+twatchs3.menu.PartitionScheme.rainmaker.upload.maximum_size=3145728
```

Comment:
Please fix this RainMaker partition or add the 8MB one instead of this -> https://github.com/espressif/arduino-esp32/pull/10046